add instruction to disable log anomaly detection (#22017)
DataDog/documentation: content/en/watchdog/alerts/_index.md

Watchdog requires some data to establish a baseline of expected behavior. For log anomalies, the minimum history is 24 hours. 
Watchdog starts finding anomalies after the minimum required history is available, and Watchdog improves as history grows. Best performances are obtained with six weeks of history. 

> #### Disabling Log anomaly detection

Reviewer: ```suggestion
#### Disabling log anomaly detection
```

> To disable Log Anomaly detection, go to the [Log Management pipeline page][4] and turn off the toogle for Log Anomalies.

Reviewer: ```suggestion
To disable log anomaly detection, go to the [Log Management pipeline page][4] and click the Log Anomalies toggle.
```

[1]: https://app.datadoghq.com/watchdog
[2]: /monitors/types/watchdog/
[3]: /watchdog/insights?tab=logmanagement#explore-insights

> [4]: /logs/log_configuration/

Reviewer: ```suggestion
[4]: https://app.datadoghq.com/logs/pipelines
```
